x-pack/plugins/fleet/server/services/epm/archive/storage.ts

```diff
@@ -224,24 +224,19 @@ export const getEsPackage = async (
       );
       const dataStreamManifest = safeLoad(soResDataStreamManifest.attributes.data_utf8);
       const {
-        title: dataStreamTitle,
-        release,
         ingest_pipeline: ingestPipeline,
-        type,
         dataset,
         streams: manifestStreams,
+        ...dataStreamManifestProps
```

Comment:
See the [first commit message](https://github.com/elastic/kibana/pull/96063/commits/4176e8f066324acb75ccf8f9e37a9970a7ab0a63)